### PR TITLE
Added URL to pydocstyle job for data-model

### PR DESCRIPTION
**Pull request by tisnik**

### CODE REVIEW

**Comment by tisnik** [comment]
[test]

**Comment by tisnik** [comment]
[test]

File: dashboard/config.ini [hunk 1]
```diff
@@ -10,6 +10,7 @@ pydoc_job=devtools-fabric8-analytics-common-fabric8-analytics-pydoc
 build_job=devtools-fabric8-analytics-data-model-f8a-build-master
 test_job=devtools-fabric8-analytics-data-model-fabric8-analytics
 pylint_job=devtools-fabric8-analytics-data-model-fabric8-analytics-pylint
+pydoc_job=https://ci.centos.org/job/devtools-fabric8-analytics-jobs-fabric8-analytics-pydoc
```
Comment (jpopelka) [inline dashboard/config.ini]: I'd say `pydoc_job=devtools-fabric8-analytics-data-model-fabric8-analytics-pydoc`

Comment (tisnik) [reply dashboard/config.ini]: you are right, bad /me

**Pull request merged**